dev_guide/builds.adoc

```diff
@@ -1426,6 +1432,96 @@ Configuration change triggers currently only work when creating a new
 able to launch a build whenever a `*BuildConfig*` is updated.
 ====
 
+[[build-hooks]]
+
+== Build Hooks
+
+Build hooks can be used as a part of the build flow to run tests inside a
```

Comment:
actually i think i did start like that and somehow ended up with tests. will redo though.